src/Austria/Mews.Fiscalizations.Austria/Mews.Fiscalizations.Austria.csproj

```diff
@@ -24,9 +24,21 @@
       <Pack>True</Pack>
       <PackagePath></PackagePath>
     </None>
+    <None Include="..\..\..\packageIcon.png">
```

Comment:
@KaliCZ I used an old icon we had on the previous repos, i didn't want to copy something from the internet due to copyrights (if its worth, we can ask a designer in Mews to create a simple icon we can use 😄 )